gcloud/logging/logger.py

```diff
@@ -298,7 +298,7 @@ def list_entries(self, projects=None, filter_=None, order_by=None,
                   more entries can be retrieved with another call (pass that
                   value as ``page_token``).
         """
-        log_filter = 'logName:%s' % (self.name,)
+        log_filter = 'logName=%s' % (self.full_name,)
```

Comment:
Do I understand correctly: the original version "worked" because the two errors (substring operator + short name) cancelled out?